Description: Upstream changes introduced in version 1.5.3-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 glew (1.5.3-2) unstable; urgency=low
 .
   * add glew.pc to package (Closes: #582546)
   * introduce glewmx.pc
 .
 The person named in the Author field signed this changelog entry.
Author: Joost Yervante Damad <andete@debian.org>
Bug-Debian: http://bugs.debian.org/582546

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ glew-1.5.3/glewmx.pc
@@ -0,0 +1,10 @@
+prefix=/usr
+exec_prefix=/usr/bin
+libdir=/usr/lib
+includedir=/usr/include/GL
+
+Name: glewmx
+Description: The OpenGL Extension Wrangler library (mx)
+Version: 1.5.3
+Cflags: -I${includedir}
+Libs: -L${libdir} -lGLEWmx
